pkg/receive/limiter_config.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package receive
+
+import (
+	"github.com/thanos-io/thanos/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// RootLimitsConfig is the root configuration for limits.
+type RootLimitsConfig struct {
```

Comment:
Why not just `LimitsConfig`? 🙂

Reply:
Too many things end up being called by almost the same name and it gets confusing over time. This is already a result of many iterations on the naming. 😅

Reply:
<img src="https://media1.giphy.com/media/l4FGrYKtP0pBGpBAY/giphy.gif"/>